components/coding/Carousel.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+
+export type CarouselProp = {};
+
+export const Carousel: React.FC<CarouselProp> = () => {
```

Comment:
awesome! Can we add this to our storybook as well. This is going to be a really great re-usable component to use in any future lessons.